--- libshout-2.2/src/Makefile.am.orig	2006-01-03 20:22:17.000000000 +0100
+++ libshout-2.2/src/Makefile.am	2006-01-10 21:53:13.264391320 +0100
@@ -26,7 +26,7 @@
 AM_CFLAGS = @XIPH_CFLAGS@
 
 libshout_la_LIBADD = net/libicenet.la timing/libicetiming.la avl/libiceavl.la\
-		httpp/libicehttpp.la $(MAYBE_THREAD_LIB) $(THEORA_LIBS) $(VORBIS_LIBS)
+		httpp/libicehttpp.la $(MAYBE_THREAD_LIB) $(THEORA_LIBS) $(VORBIS_LIBS) $(SPEEX_LIBS)
 
 INCLUDES = -I$(top_builddir)/include
 
