--- libshout-2.4.2/src/tls.c.orig	2019-05-28 17:19:44.766778317 +0200
+++ libshout-2.4.2/src/tls.c	2019-05-28 17:21:49.872767226 +0200
@@ -90,7 +90,11 @@
 
     SSL_library_init();
     SSL_load_error_strings();
+#if OPENSSL_VERSION_NUMBER < 0x10100000
     SSLeay_add_all_algorithms();
+#else
+    OpenSSL_add_all_algorithms();
+#endif
     SSLeay_add_ssl_algorithms();
 
     meth = TLSv1_client_method();
